hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/rmadmin/TestFederationRMAdminInterceptor.java

```diff
@@ -120,7 +120,7 @@ public void setUp() {
     // Initialize facade & stateSore
     stateStore = new MemoryFederationStateStore();
     stateStore.init(this.getConf());
-    facade = FederationStateStoreFacade.getInstance();
+    facade = FederationStateStoreFacade.getInstance(this.getConf());
```

Comment:
Let's be consistent with using this or not.

Reply:
I will read the code carefully to ensure the uniformity of usage. If there is `configuration` in the usage class involved, we will use `FederationStateStoreFacade.getInstance(this.getConf())`, if there is no `configuration` in the usage class involved, we will use Will use `FederationStateStoreFacade.getInstance()`.